ThirdPartyNotices.txt

```diff
@@ -0,0 +1,684 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+The Visual Studio Application Insights SDK for .NET Web Applications source code repository includes third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
+
+
+1	Antlr (https://github.com/antlr/antlrcs)
+2	AspNet.ScriptManager.bootstrap (https://www.nuget.org/packages/AspNet.ScriptManager.bootstrap/)
+3	AspNet.ScriptManager.jQuery (https://www.nuget.org/packages/AspNet.ScriptManager.jQuery/)
+4	bootstrap (http://getbootstrap.com/)
+5	DotNetOpenAuth.AspNet (http://dotnetopenauth.net/)
+6	DotNetOpenAuth.Core (http://dotnetopenauth.net/)
+7	DotNetOpenAuth.OAuth.Consumer (http://dotnetopenauth.net/)
+8	DotNetOpenAuth.OAuth.Core (http://dotnetopenauth.net/)
+9	DotNetOpenAuth.OpenId.Core (http://dotnetopenauth.net/)
+10	DotNetOpenAuth.OpenId.RelyingParty (http://dotnetopenauth.net/)
+11	gitlink (https://github.com/GitTools/GitLink)
+12	jQuery (http://jquery.com/)
+13	jQuery.UI.Combined (http://jqueryui.com/)
+14	jQuery.Validation (http://bassistance.de/jquery-plugins)
+15	knockoutjs (http://knockoutjs.com/)
+16	Modernizr (https://modernizr.com/)
+17	Newtonsoft.Json (http://www.newtonsoft.com/json)
+18	OpenCover (https://github.com/opencover/opencover)
+19	Owin (https://github.com/owin-contrib/owin-hosting/)
+20	Respond (https://github.com/scottjehl/Respond/)
+21	Rx-Core (https://github.com/Reactive-Extensions/Rx.NET)
+22	Rx-Interfaces (https://github.com/Reactive-Extensions/Rx.NET)
+23	Rx-Linq (https://github.com/Reactive-Extensions/Rx.NET)
+24	StyleCop.MSBuild (https://github.com/adamralph/stylecop-msbuild)
+25	System.Spatial (http://odata.github.io/)
+26	WebGrease (http://webgrease.codeplex.com/)
+27	WindowsAzure.Storage (https://azure.microsoft.com/en-us/documentation/services/storage/)
```

Comment:
It did not start with Microsoft.*? I thought we still need to add a notice for that.